lib/gateway/make-resolver.js

```diff
@@ -454,7 +463,8 @@ function makeResolver ({ service, createOperation, transformData, isQuery, isRef
     return reply[kEntityResolvers][`${service.name}Entity`]({
       query,
       variables,
-      originalRequestHeaders: reply.request.headers
+      originalRequestHeaders: reply.request.headers,
+      context
     }).then(transformData)
```

Comment:
please change then to `await`, `makeResolver` is now an async function (also a few lines above)

Reply:
sure!